apps/web/src/components/ExplorerDetails.tsx

```diff
@@ -55,6 +56,7 @@ export function ExplorerDetails() {
 
   return (
     <div className="sm:fle flex w-full flex-wrap items-center justify-center gap-2 align-middle text-xs text-contentSecondary-light dark:text-contentSecondary-dark sm:h-4 sm:justify-start">
+      <LatestGasPrice />
```

Comment:
Can we refactor this to include it as part of the `explorerDetailsItems` array?

Reply:
Take a look at how it's done for the "Last synced slot" item

Reply:
Done